src/typings/Types.ts

```diff
@@ -181,6 +181,16 @@ export interface Edit {
   isReplace?: boolean;
 }
 
+/** Represents a custom tokenizer for a language */
+
+export interface DefaultLanguageTokenizer {
```

Comment:
Shouldn't this be named something like
```suggestion
export interface LanguageTokenizerComponents {
```